railties/lib/rails/test_unit/testing.rake

```diff
@@ -23,13 +23,14 @@ namespace :test do
 
   task run: %w[test]
 
-  desc "Run tests quickly, but also reset db"
+  desc "Runs tests quickly, but also reset db"
```

Comment:
This runs all tests except `test:system` so a better description might be something like:
```suggestion
  desc "Resets the database, then runs `test`"
```